ui/client/components/graph/GraphContext.tsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable i18next/no-literal-string */
+import React, {createContext, PropsWithChildren, useContext} from "react"
+import Graph from "./Graph"
+
+const GraphContext = createContext<() => Graph>(null)
```

Comment:
Maybe we could extract () => Graph as separate type?

Reply:
what will we get from it?